python/ray/util/metrics.py

```diff
@@ -15,6 +16,24 @@
 logger = logging.getLogger(__name__)
 
 
+def _validate_metric_name(name: str):
+    METRIC_NAME_RE = re.compile(r"^[a-zA-Z_:][a-zA-Z0-9_:]*$")
+
+    if len(name) == 0:
+        raise ValueError("Empty name is not allowed. Please provide a metric name.")
+
+    if not METRIC_NAME_RE.match(name):
+        """
```

Comment:
Can you please attach this comment to regex line?

Reply:
Done